docs-site/src/index.ts

```diff
@@ -18,6 +18,7 @@ export default {
 
 const redirect_lookup: Record<string, string> = {
   '/common_tools': '/common-tools/',
+  '/testing-evals': '/testing/',
```

Comment:
is it expected that https://d0538eef-pydantic-ai-previews.pydantic.workers.dev/testing-evals/ doesn't work?

Reply:
yes, we use the worker from main.